recipes/image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.5.0" %}
+
+package:
+    name: image-meta-tag
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/SciTools-incubator/image-meta-tag
+    git_tag: {{ version }}
+    sha1: f0b204bdb789cb8c8efb5cf9b30f9677684f0728
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python >=2.7,<3
```

Comment:
Rather than specifying the python version here, please use the `skip` tag in the `build`:

http://conda-forge.github.io/docs/meta.html#excluding-a-platform
https://conda.io/docs/building/meta-yaml.html#preprocessing-selectors